onnxruntime/core/providers/qnn/builder/qnn_utils.cc

```diff
@@ -570,6 +578,27 @@ Status Quantize(const double double_value,
   return Status::OK();
 }
 
+const char* GetQnnErrorMessage(const QNN_INTERFACE_VER_TYPE& qnn_interface, Qnn_ErrorHandle_t qnn_error_handle) {
+  // From QNN SDK: The memory is statically owned and should not be freed by the caller.
+  const char* error_msg = nullptr;
```

Comment:
Should we return a copy by returning an std::string instead to avoid user being confused on whether or not to free the memory?

Reply:
Or std::string_view to avoid the copy?